cyclonedx_py/parser/environment.py

```diff
@@ -75,18 +75,18 @@ def __init__(self, use_purl_bom_ref: bool = False) -> None:
                 # Therefore, just go with a named license.
                 c.licenses.add(LicenseChoice(license_=License(license_name=i_metadata['License'])))
 
-            if 'Classifier' in i_metadata:
-                for classifier in i_metadata['Classifier']:
-                    # Trove classifiers - https://packaging.python.org/specifications/core-metadata/#metadata-classifier
-                    # Full list: https://pypi.python.org/pypi?%3Aaction=list_classifiers
-                    if str(classifier).startswith('License :: OSI Approved :: '):
-                        c.licenses.add(LicenseChoice(license_=License(
-                            license_name=str(classifier).replace('License :: OSI Approved :: ', '').strip()
-                        )))
-                    elif str(classifier).startswith('License :: '):
-                        c.licenses.add(LicenseChoice(license_=License(
-                            license_name=str(classifier).replace('License :: ', '').strip()
-                        )))
+            classifiers = [value for (key, value) in i_metadata._headers if key == 'Classifier']  # type: ignore
```

Comment:
:x: `._headers` is a non-public property.
therefore, it should not be used. Please find the official/public getter for it.

Reply:
Solved. I found and use `i_metadata.get_all("Classifier")`.